routes/consumption_history.py

```diff
@@ -0,0 +1,107 @@
+import os
+from datetime import datetime
+import logging
+from flask import Blueprint, jsonify, request
+from flask import current_app as app
+from discovergy.discovergy import Discovergy
+
+
+logger = logging.getLogger(__name__)
+IndividualConsumptionHistory = Blueprint('IndividualConsumptionHistory',
+                                         __name__)
+GroupConsumptionHistory = Blueprint('GroupConsumptionHistory', __name__)
+
+
+def read_parameters():
+    """ Use the given parameters. """
+
+    start = round(datetime.combine(datetime.now(),
+                                   datetime.min.time()).timestamp() * 1e3)
+    begin = request.args.get('begin', default=start, type=int)
+    end = request.args.get('end', default=None, type=int)
+    tics = request.args.get('tics', default='one_hour', type=str)
```

Comment:
why not an enum?

Reply:
To return a list, you mean? AFAIK, in Python you rather return tuples if possible (their members are numbered just the same).